src/Sql/dbo/Tables/TaxRate.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE [dbo].[TaxRate] (
+    [Id]                VARCHAR(40)         NOT NULL        PRIMARY KEY,
+    [Country]           VARCHAR(50)         NOT NULL,
+    [State]             VARCHAR(2)          NULL,
+    [PostalCode]        VARCHAR(10)         NOT NULL,
+    [Rate]              DECIMAL(5,2)        NOT NULL,
+    [Active]        BIT                 NOT NULL
+);
+GO
+
+ALTER TABLE [dbo].[TaxRate]
+ADD CONSTRAINT Unique_Country_PostalCode
```

Comment:
Have a look at User table. I think we prefix unique constraints someway.

Reply:
I changed this to an index b/c a flat constraint wasn't accounting for the fact that only `Active = 1` records need to have a unique location. Still used the Users table for the naming convention for the Index, though 😄 